media/css/firefox/private_browsing/private-browsing.less

```diff
@@ -0,0 +1,489 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../../sandstone/lib.less";
+
+/*-------------------------------------------------------------------------*/
+// Common
+
+#wrapper {
+    border-top: 2px solid #fff;
+    background-color: #0095dd;
+    margin-top: -162px; // family nav
+    padding-bottom: 0;
+    width: 100%;
+}
+.container {
+    .border-box();
+    .font-size(22px);
+    margin: 0 auto;
+    padding-top: 90px;
+    padding-bottom: 90px;
+    width: @widthDesktop;
+}
+h2, h3 {
+    text-align: center;
+}
+h1 {
+    color: #fff;
+    .open-sans-light();
+    .font-size(66px);
+    line-height: 1.1;
+    text-shadow: none;
+    margin-bottom: 40px;
+}
+h2 {
+    color: #652562;
+    .open-sans();
+    .font-size(48px);
+    margin-bottom: 60px;
+}
+h3 {
+    color: #652562;
+    .font-size(25px);
+    line-height: 1.2;
+    margin: @baseLine auto;
+}
+
+p {
+    color: @mozIDGrey;
+    .open-sans-light();
+    .font-size(20px);
+
+}
+
+ul {
+    list-style-type: none;
+    margin: 0;
+}
+
+.footer {
+    margin: 45px auto 0 auto ;
+    text-align: center;
+  
+    a {
+        display: none;
+    }
+    a::after {
+        content:" »";
+        white-space: nowrap;
+    }
+    strong {
+        margin-bottom: 50px;
+        display: block;
+    }
+    small {
+        color: #56565a;
+        .open-sans-italic();
+        display: block;
+        margin: auto;
+    }
+    .desktop-link {
+        display: block;
+    }
+}
+
+.button-flat-dark {
+    .open-sans-light;
+    .font-size(24px);
+    padding: 12px @baseLine;
+    .transition(background-color 250ms linear 0s);
+}
+
+.download-buttons  {
+    p {
+        color: #fff;
+        .font-size(22px);
+    } 
+    .download-button small.download-other {
+        color: #71C4EA;
+        a {
+            color: inherit;
+        }
+    }
+}
+
+/*-------------------------------------------------------------------------*/
+// Main Sections 
+
+.tracking-protection,
+.when-to-use,
+.control-center {
+    h2 {
+        margin-bottom: (@baseLine * 3);
+        padding: 0 30px;
+    }
+    p {
+        margin: @baseLine auto 0 auto;
+        padding: 0 15px;
+    }
+}
+
+.intro {
+    margin-top: 50px;
+
+    header {
+        width: 490px;
+        padding-top: 80px;
+        float: left;
+    }
+    .critter-animation {
+        float: right;
+        margin: 30px -85px 0 0; 
+    }
+}
+
+.tracking-protection {
+    background-color: #fff;
+  
+    h3 {
+        .open-sans-extrabold();
+        .font-size(30px);
+        margin: 10px auto; 
+    }
+    .blurb {
+        background: url('/media/img/firefox/private-browsing/sprite-tracking-protection.png') no-repeat;
+        background: linear-gradient(transparent, transparent), url('/media/img/firefox/private-browsing/sprite-tracking-protection.svg');
```

Comment:
depends on which IE version(s) this is for. It is only supported by IE10+ and the mixin adds the `-ms-filter` part for older versions of IE that does not support `linear-gradient`.

http://caniuse.com/#search=linear-gradient